lib/handlers/quote/util/quote-provider-traffic-switch-configuration.ts

```diff
@@ -8,106 +9,244 @@ export type QuoteProviderTrafficSwitchConfiguration = {
-      return {
-        switchExactInPercentage: 100,
-        samplingExactInPercentage: 0,
-        switchExactOutPercentage: 100,
-        samplingExactOutPercentage: 0,
-      } as QuoteProviderTrafficSwitchConfiguration
+      switch (protocol) {
+        case Protocol.MIXED:
+        case Protocol.V4:
+          return {
+            switchExactInPercentage: 0,
+            samplingExactInPercentage: 0,
+            switchExactOutPercentage: 0,
+            samplingExactOutPercentage: 0,
+          } as QuoteProviderTrafficSwitchConfiguration
+        default:
+          // Celo RPC eth_call traffic is about 1/100 of mainnet, so we can shadow sample 10% of traffic
+          return {
+            switchExactInPercentage: 100,
+            samplingExactInPercentage: 0,
+            switchExactOutPercentage: 100,
+            samplingExactOutPercentage: 0,
+          } as QuoteProviderTrafficSwitchConfiguration
+      }
     case ChainId.AVALANCHE:
-      // Avalanche RPC eth_call traffic is about 1/100 of mainnet, so we can shadow sample 10% of traffic
-      return {
-        switchExactInPercentage: 100,
-        samplingExactInPercentage: 0,
-        switchExactOutPercentage: 100,
-        samplingExactOutPercentage: 0,
-      } as QuoteProviderTrafficSwitchConfiguration
+      switch (protocol) {
+        case Protocol.MIXED:
+        case Protocol.V4:
+          return {
+            switchExactInPercentage: 0,
+            samplingExactInPercentage: 0,
+            switchExactOutPercentage: 0,
+            samplingExactOutPercentage: 0,
+          } as QuoteProviderTrafficSwitchConfiguration
+        default:
+          // Avalanche RPC eth_call traffic is about 1/100 of mainnet, so we can shadow sample 10% of traffic
+          return {
+            switchExactInPercentage: 100,
+            samplingExactInPercentage: 0,
+            switchExactOutPercentage: 100,
+            samplingExactOutPercentage: 0,
+          } as QuoteProviderTrafficSwitchConfiguration
+      }
     // worldchain and astrochain sepolia don't have the view-only quoter yet, so we can shadow sample 0.1% of traffic
     case ChainId.WORLDCHAIN:
     case ChainId.ASTROCHAIN_SEPOLIA:
-      return {
-        switchExactInPercentage: 0.0,
-        samplingExactInPercentage: 0.1,
-        switchExactOutPercentage: 0.0,
-        samplingExactOutPercentage: 0.1,
-      } as QuoteProviderTrafficSwitchConfiguration
+      switch (protocol) {
+        case Protocol.MIXED:
+        case Protocol.V4:
+          return {
+            switchExactInPercentage: 0,
+            samplingExactInPercentage: 0,
+            switchExactOutPercentage: 0,
+            samplingExactOutPercentage: 0,
+          } as QuoteProviderTrafficSwitchConfiguration
+        default:
+          return {
+            switchExactInPercentage: 0.0,
+            samplingExactInPercentage: 0.1,
+            switchExactOutPercentage: 0.0,
+            samplingExactOutPercentage: 0.1,
```

Comment:
should this be 0.1 or 1?

Reply:
0.1, meaning 0.1%. it's the same as it has been at 0.1% https://github.com/Uniswap/routing-api/pull/868/files/a86acba2a09920a8377cbef7e6de2a636002a62a#diff-a714608191ad685cf372e11933218d41d038498910b953547a18d4266666bee5R230